src/regression_tests/regression_test.test.ts

```diff
@@ -0,0 +1,138 @@
+import fetchMock from 'jest-fetch-mock';
+import {
+  createUser,
+  getBalance,
+  getEnvVariables,
+  getPaymailDomain,
+  getTransactions,
+  RegressionTestConfig,
+  RegressionTestUser,
+  removeRegisteredPaymail,
+  sendFunds,
+} from './utils';
+
+const MINIMAL_FUNDS_PER_TRANSACTION = 2;
+// Two minute timeout for transaction tests
+const TEST_TIMEOUT_MS = 120000;
+const ADMIN_XPRIV =
+  'xprv9s21ZrQH143K3CbJXirfrtpLvhT3Vgusdo8coBritQ3rcS7Jy7sxWhatuxG5h2y1Cqj8FKmPp69536gmjYRpfga2MJdsGyBsnB12E19CESK';
+const ADMIN_XPUB =
+  'xpub661MyMwAqRbcFgfmdkPgE2m5UjHXu9dj124DbaGLSjaqVESTWfCD4VuNmEbVPkbYLCkykwVZvmA8Pbf8884TQr1FgdG2nPoHR8aB36YdDQh';
+
+let paymailDomainInstanceOne: string = '';
+let paymailDomainInstanceTwo: string = '';
+let userOne: RegressionTestUser;
+let userTwo: RegressionTestUser;
+let rtConfig: RegressionTestConfig;
+
+beforeAll(() => {
+  fetchMock.disableMocks();
+  rtConfig = getEnvVariables();
+  expect(rtConfig).toBeDefined();
+});
+
+afterAll(async () => {
+  if (userOne) {
+    await expect(removeRegisteredPaymail(userOne.paymail, rtConfig.clientOneURL, ADMIN_XPRIV)).resolves.not.toThrow();
+  }
+
+  if (userTwo) {
+    await expect(removeRegisteredPaymail(userTwo.paymail, rtConfig.clientTwoURL, ADMIN_XPRIV)).resolves.not.toThrow();
+  }
+});
+
+describe('TestRegression', () => {
+  describe('Initialize Shared Configurations', () => {
+    test('Should get sharedConfig for instance one', async () => {
+      paymailDomainInstanceOne = await getPaymailDomain(ADMIN_XPRIV, rtConfig.clientOneURL);
+      expect(paymailDomainInstanceOne).not.toBe('');
+    });
+
+    test('Should get sharedConfig for instance two', async () => {
+      paymailDomainInstanceTwo = await getPaymailDomain(ADMIN_XPRIV, rtConfig.clientOneURL);
```

Comment:
Should't be instance `"Two"`?
```suggestion
      paymailDomainInstanceTwo = await getPaymailDomain(ADMIN_XPRIV, rtConfig.clientTwoURL);
```

Reply:
Oh right, a typo 🙈 They are the same url but you are correct. Changed.